.github/CODEOWNERS

```diff
@@ -68,141 +99,141 @@
 
 # PRLabel: %Event Hubs
-/sdk/eventhubs/microsoft-azure-eventhubs-eph/       @JamesBirdsall @sjkwak
+/sdk/eventhubs/microsoft-azure-eventhubs-eph/        @JamesBirdsall @sjkwak
 
 # PRLabel: %Event Hubs
-/sdk/eventhubs/microsoft-azure-eventhubs-extensions/  @JamesBirdsall @sjkwak
+/sdk/eventhubs/microsoft-azure-eventhubs-extensions/ @JamesBirdsall @sjkwak
 
 # PRLabel: %Event Hubs
-/sdk/eventhubs/microsoft-azure-eventhubs/           @JamesBirdsall @sjkwak
+/sdk/eventhubs/microsoft-azure-eventhubs/            @JamesBirdsall @sjkwak
 
 # PRLabel: %Cognitive - Form Recognizer
-/sdk/formrecognizer/                                @samvaity @mssfang
+/sdk/formrecognizer/                                 @samvaity @mssfang
 
 # PRLabel: %Cognitive - Metrics Advisor
-/sdk/metricsadvisor/                                @samvaity @anuchandy
+/sdk/metricsadvisor/                                 @samvaity @anuchandy
 
 # PRLabel: %Cognitive - Anomaly Detector
-/sdk/anomalydetector/                               @conhua @mengaims @juaduan @moreOver0
+/sdk/anomalydetector/                                @conhua @mengaims @juaduan @moreOver0
 
 # PRLabel: %Azure.Identity
-/sdk/identity/                                      @g2vinay @schaabs @joshfree @billwert
+/sdk/identity/                                       @g2vinay @schaabs @joshfree @billwert
 # PRLabel: %azure-spring
-/sdk/identity/azure-identity-extensions/           @backwind1233 @chenrujun @hui1110 @netyyyy @saragluna @stliu @yiliuTo @moarychan @fangjian0423
+/sdk/identity/azure-identity-extensions/             @backwind1233 @chenrujun @hui1110 @netyyyy @saragluna @stliu @yiliuTo @moarychan @fangjian0423
 
 # PRLabel: %KeyVault
-/sdk/keyvault/                                      @vcolin7 @g2vinay @samvaity
+/sdk/keyvault/                                       @vcolin7 @g2vinay @samvaity
 # PRLabel: %azure-spring
-/sdk/keyvault/azure-security-keyvault-jca           @backwind1233 @chenrujun @hui1110 @netyyyy @saragluna @stliu @yiliuTo @moarychan @fangjian0423
+/sdk/keyvault/azure-security-keyvault-jca/           @backwind1233 @chenrujun @hui1110 @netyyyy @saragluna @stliu @yiliuTo @moarychan @fangjian0423
 
-/sdk/loganalytics/microsoft-azure-loganalytics/     @divyajay @alongafni
+/sdk/loganalytics/microsoft-azure-loganalytics/      @divyajay @alongafni
 
 # PRLabel:  %Monitor - Exporter
-/sdk/monitor/azure-monitor-opentelemetry-exporter   @trask @ramthi @heyams @jeanbisutti
+/sdk/monitor/azure-monitor-opentelemetry-exporter/   @trask @ramthi @heyams @jeanbisutti
 
 # PRLabel:  %Monitor
-/sdk/monitor/azure-monitor-query                    @srnagar @lmolkova @ki1729
+/sdk/monitor/azure-monitor-query*/                  @srnagar @lmolkova @ki1729
 
 # PRLabel:  %Monitor
-/sdk/monitor/azure-monitor-ingestion                @srnagar @lmolkova @ki1729
+/sdk/monitor/azure-monitor-ingestion*/              @srnagar @lmolkova @ki1729
```

Comment:
@srnagar @lmolkova @ki1729 I want to bring your attention to these two changes. This means you are also taking ownership of contents of 
`/sdk/monitor/azure-monitor-query-perf/`
and
`/sdk/monitor/azure-monitor-ingestion-perf/`

Previously these directories matched to `/sdk/` owned by `joshfree,srnagar,anuchandy,conniey,lmolkova,jonathangiles`

If this is undesired behavior, please do let me know, I'll rever the change.

CC @weshaggard @jsquire